src/components/form/inputs/TextInputNew2.tsx

```diff
@@ -76,8 +76,8 @@ export default function TextInputNew({
         value={textValue}
         onFocus={handleFocus}
         onBlur={() => {
-          setCharacterError(false);
-          setColor && setColor(false, name);
+          // setCharacterError(false);
+          // setColor && setColor(false, name);
```

Comment:
please delete instead of commenting, do any other components need this functionality since this is a common component